docs/reks/introduction.rst

```diff
@@ -61,28 +59,34 @@ following input that follows::
     VerbosityLevel = 1
   }
 
-Note that the ``REKS = SSR22`` block is a separate block like ``ExcitedStates`` block,
-which means it is not included in the ``Hamiltonian`` block. In REKS calculation
-``Hamiltonian`` block only controls a detailed Hamiltonian setting such as range-separated
-hybrid functional or dispersion corrections. The three singlet states can be generated
-from (2,2) active space. These are perfectly spin-paired singlet state (PPS), open-shell
-singlet state (OSS) and doubly excited singlet state (DES). Combining these singlet states,
-we can determine the singlet states to be included in minimized energy functional. In
-single-state REKS case we treats only one singlet state and it becomes PPS state. Thus,
-you can only include the PPS state into ``Functional`` block in ``Energy`` block.
+Note that the ``REKS = SSR22`` block is a separate environment in the
+calculation, so is not included in the ``Hamiltonian`` block. In a REKS
+calculation, the ``Hamiltonian`` only controls detailed Hamiltonian settings
+such as use of a range-separated hybrid functional or dispersion
+corrections. Three singlet states can be generated from a (2,2) active
+space. These are the single particle perfectly spin-paired singlet state (PPS),
+an open-shell singlet state (OSS) and the doubly excited singlet state
+(DES). Combining these singlet states, we can find the many-particle singlet
+state which minimises the energy functional. In the single-state REKS case we
```

Comment:
Three singlet states (PPS, OSS, DES) are many-body state, thus I think `single particle state` is not proper expression.